src/lib/utils.ts

```diff
@@ -0,0 +1,29 @@
+const loadMathJax = async (): Promise<boolean> => {
+  const maybeMoodleYUI = window.Y
+
+  if (maybeMoodleYUI === undefined) {
+    // TODO: Load MathJax manually to support things like preview outside of
+    // Moodle
+    return false
```

Comment:
I hit this once and I just observed @mjoss17 experience the same: I believe that sometimes we hit this code path even when we're in Moodle and math ends up not rendering. It must be due to some async loading and ensuing race condition. I'll dig into it a bit more.

Reply:
I wasn't able to reproduce this, but after some code inspection: 1) I believe my initial hunch about this code path being taken was wrong (our script is defer'd and Moodle is loading globals like `M` and `Y` through in-line / synchronously loaded JS); 2) I found where a definite async issue that can occur between MathJax and the `ContentLoader` completing. I just missed this when writing the code, and it happened to randomly work most of the time despite the flawed logic. In any case, [this commit](https://github.com/openstax/k12-apps-raise/pull/1/commits/209a520e165b0cb5c7563dbf1ebcd2e63104c109) should fix things.